lib/performance_data/statistics.rb

```diff
@@ -1,39 +1,76 @@
 require 'json'
 require 'gds_api/performance_platform/data_out'
+require 'performance_data/metrics'
 
 module PerformanceData
   class Statistics
     ALLOWED_DOC_TYPES = %w('smart_answer', 'simple_smart_answer').freeze
 
     attr_accessor :data_out, :slug, :part_urls, :is_multipart,
```

Comment:
I wonder if we could change `is_multipart` to be the more idiomatic `multipart?`.  Not just in this object, but throughout the views etc too?